thoughts-of-a-rustacean-learning-go.md

```diff
@@ -0,0 +1,244 @@
+<!--
+headline: Thoughts of a Rustacean learning Go
+description: the experiences of a Rustacean learning Go
+author: Manish Goregaokar
+datePublished: 2015-04-24
+twitter: ManishEarth
+github: manishearth
+-->
+
+// Might want to mention that this is a crosspost
```

Comment:
I think that since we'll be making edits to the article for polyglotweekly.com, we probably don't need to explicitly mention that it's a cross post (unless you'd like to).